datadog_checks_dev/README.md

```diff
@@ -10,10 +10,17 @@
 This is the developer toolkit designed for use by any [Agent-based][5] check or
 integration repository.
 
+## Prerequisites
+
+* Python 2.7 and Python 3.7+ need to be available on your system.
```

Comment:
```suggestion
* Python 2.7 or Python 3.7+ need to be available on your system.
```

Reply:
If some of the checks require Python 3.6+ (as noted below), then shouldn't that (or 3.7) be a hard requirement?

Reply:
Can we just say 3.7+ then?

Reply:
```markdown
* Python 3.7+ needs to be available on your system. Python 2.7 is optional.
```
?

Reply:
> Can we just say 3.7+ then?

Isn't both 2.7 and 3.7+ needed to run all the tests ? 

e.g. https://github.com/DataDog/integrations-core/blob/master/yarn/tox.ini#L4-L5

Reply:
> Isn't both 2.7 and 3.7+ needed to run all the tests ?

Yes, but only 3.6+ is _necessary_, since the 2.7 checks _can_ be skipped.

Hooray for EOL and transitional cases!